packages/interfaces/src/waku.ts

```diff
@@ -18,7 +18,7 @@ export interface Waku {
 
   connectionManager: IConnectionManager;
 
-  dial(peer: PeerId | Multiaddr, protocols?: Protocols[]): Promise<Stream>;
+  dial(peer: PeerId | MultiaddrInput, protocols?: Protocols[]): Promise<Stream>;
```

Comment:
fix for discrepancy in types - this prevents us from doing `.dial(multiAddrStr)` in TS